src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -138,6 +162,43 @@ export const TranscriptsListItemInfo = (
     return urlFor.browser({ genomeId: genomeId, focus: focusIdForUrl });
   };
 
+  const chevronClassForDownload = classNames(styles.chevron, {
+    [styles.chevronUp]: props.expandDownload
+  });
+
+  const chevronClassForMoreInfo = classNames(styles.chevron, {
+    [styles.chevronUp]: props.expandMoreInfo
+  });
```

Comment:
Both chevron classes unused?

Why didn't eslint pick it up? Is it broken on your machine?

Reply:
Thats very strange. I have just tried committing some changes and leaving these unused classes and eslint did pick them up. 
I think i know what happen, its when i did the branch rebase with dev and after resolving conflicts, i did  the commit and i dont think eslint works during the rebase. I cant remember seeing the eslint procedures. 

I have now removed them.